bundles/extensions/i18n/src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -226,6 +257,14 @@ public void handleEvent(final org.osgi.service.event.Event event) {
         }
     }
 
+    private void discardBundlesWithLocalePrefix(String localePrefix) {
+        for (JcrResourceBundle bundle : resourceBundleCache.values()) {
+            if (bundle.getLocale().toString().startsWith(localePrefix)) {
+                scheduleReloadBundle(bundle);
```

Comment:
Actually scheduleReloadBundle already takes care of clearing dependent cached bundles